anchore_engine/services/policy_engine/engine/feeds/sync.py

```diff
@@ -64,8 +64,11 @@ def instance(cls):
     @staticmethod
     def update_counts():
         for feed in get_all_feeds_detached():
-            f = feed_instance_by_name(feed.name)
-            f.update_counts()
+            try:
+                f = feed_instance_by_name(feed.name)
```

Comment:
Is there where it's failing in the upgrade path? I'm guessing all other changes are necessary to support flush ops on the feed

Reply:
Yes, this would have fixed upgrade without the other changes, but leaves the data in limbo without the other changes.